app/addons/permissions/container/PermissionsContainer.js

```diff
@@ -24,14 +24,14 @@ import {
 } from '../reducers';
 
 
-const mapStateToProps = (state) => {
+const mapStateToProps = ({permissions}) => {
   return {
```

Comment:
Permission are now named space in the reducer so with need to pick permissions from the whole state